apps/server/src/modules/task/repo/task.repo.ts

```diff
@@ -21,69 +20,34 @@ export class TaskRepo {
 	 * @returns
 	 */
 	async findAll(parentIds: EntityId[], options?: IFindOptions<Task>): Promise<Counted<Task[]>> {
-		const lessons = await this.findLessons(parentIds);
+		const visibleLessons = await this.findVisibleLessons(parentIds);
```

Comment:
More important than the other picked tickets are that we move the lesson out of the repo. This is one of the highest prio tickets.

Reply:
agreed. the decision which lessons are visible to the user belong in the use case / authorization and not the repo.